app/models/concerns/host_common.rb

```diff
@@ -154,12 +154,10 @@ def crypt_root_pass
   end
 
   def param_true?(name)
-    Foreman::Deprecation.renderer_deprecation('1.17', __method__, 'host_param_true?')
```

Comment:
shouldn't this method be removed completely if it's deprecated rather then the deprecation?

Reply:
This method is deprecated only for use in templates, we can still use it internally.

Reply:
Do we? I couldn't find anywhere we do, no point in keeping around unused methods that were deprecated (unlike a few other deprecations this one wasn't only in the renderer). Also should be removed from host and hostgroup jail and tests.